source/framework/tools/inc/TRestSystemOfUnits.h

```diff
@@ -23,16 +24,16 @@
 #include <string>
 
 #ifdef REST_UnitsAdd_Caller
-#define AddUnit(name, type, scale) const double name = _AddUnit(#name, type, scale)
+#define AddUnit(name, type, scale) double name = _AddUnit(#name, type, scale)
```

Comment:
I know this is not related with this PR, but do we really need a macro for that? I think we just need a lookup table

```
const std::map REST_Units_Map<std::string, std::pair<Physical_Unit, double> >{
  {"meV", std::make_pair (REST_Units::Energy, 1e6) },
  {"keV", std::make_pair (REST_Units::Energy, 1e3) },
  ....
}
```
Then to check for a particular unit:
```
if(REST_Units_Map.find("myUnit") == REST_Units_Map.end() ) return error;//Unit not found
Physical_Unit unit =  REST_Units_Map["myUnit"].first;
double conversion = REST_Units_Map["myUnit"].second;
```

Reply:
I dont think it is a systematic practice in our code

Reply:
> I dont think it is a systematic practice in our code

I agree, in this case I think it's very well contained.

Reply:
why did you request I remove the `const` qualifier @juanangp ?

Reply:
I didn't request to remove any `const` under this PR, did I?

Reply:
> I didn't request to remove any `const` under this PR, did I?

nevermind, I got it wrong, sorry.